web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -625,9 +625,26 @@ goog.require('gn_alert');
             'related': ['parent', 'children', 'services', 'datasets']
           },
           'linkTypes': {
-            'links': ['LINK', 'kml'],
-            'downloads': ['DOWNLOAD'],
-            'layers': ['OGC:WMS', 'OGC:WFS','OGC:WMTS', 'ESRI:REST'],
+            'links': [
+              'LINK'
+            ],
+            'downloads': [
+              'WWW:DOWNLOAD',
+              'WWW:OPENDAP',
+              'WWW:FTP',
+              'KML'
+            ],
+            // 'downloadServices': [
```

Comment:
The reason for this code to be commented out is not clear.

Reply:
There is no need to remove it, it was more related to if it wouldn't be ok to uncomment it, by default.